packages/docusaurus/src/webpack/server.ts

```diff
@@ -17,11 +17,9 @@ import LogPlugin from './plugins/LogPlugin';
 
 export default function createServerConfig({
   props,
-  minify = true,
```

Comment:
Since we are not doing any optimizations in the server config I removed the `minify` option so as not to be misleading (instead this, the default values for minify-related options are defined in the base config function)